### DISCUSSION TITLE
PythonVirtualEnvOperator stopped working

### DISCUSSION THREAD

**Discussion by acutis-expeditus** [apache/airflow#39387] [General]
We have a PythonVirtualEnvOperator task that stopped working with no particular errors being logged, except the below 2 lines at the end of the log
[2024-05-02, 14:23:27 UTC] {process\_utils.py:182} INFO - Executing cmd: /tmp/venvywqr1u\_s/bin/pip install -r /tmp/venvywqr1u\_s/requirements.txt
[2024-05-02, 14:43:28 UTC] {scheduler\_job\_runner.py:1766} ERROR - Detected zombie job:

This was a task that have been running without any issues for over a month now. No changes to the airflow deployed environment or dag/task code.

**Comment by Taragolis** [comment]
> ERROR - Detected zombie job:

That is your error, other things a difficult to say without full task log, scheduler logs and worker logs

**Reply by acutis-expeditus** [reply]
Ok will try to look into scheduler logs and worker logs too
